doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -166,6 +166,32 @@ skip:
   - [Feature:CrossNamespacePodAffinity]
   - [Feature:StorageVersionAPI]
 
+### Dataplane benchmarking
+
+[Kubernetes perf-tests](https://github.com/kubernetes/perf-tests) stores
+various Kubernetes-related performance test related tools especially
+[netperf](https://github.com/kubernetes/perf-tests/tree/master/network/benchmarks/netperf)
+which benchmarks Kubernetes networking performance.
+
+As listed in [README](https://github.com/kubernetes/perf-tests/tree/master/network/benchmarks/netperf#readme),
+the 5 major network traffic paths are combination of Pod IP vs Virtual IP and
+whether the pods are co-located on the same node/VM versus a remotely located
+pod:
+  - same VM using Pod IP
+  - same VM using Cluster/Virtual IP
+  - remote VM using Pod IP
+  - remote VM using Cluster/Virtual IP
+  - same VM Pod Hairpin: Worker 2 to itself using Cluster IP
```

Comment:
I suggest to use "node" instead of "VM"

```suggestion
  - same node using Pod IP
  - same node using Cluster/Virtual IP
  - remote node using Pod IP
  - remote node using Cluster/Virtual IP
  - same node Pod Hairpin: Worker 2 to itself using Cluster IP
```

Reply:
I was hesitating between the official description and your proposal too.
Thank you for your feedback.